src/corona_hakab_model_data/default_params.json

```diff
@@ -0,0 +1,38 @@
+{
+    "population_size": 10000,
+    "total_steps": 250,
+    "initial_infected_count": 20,
+    "distributions": {
+        "latent_to_silent_days": "(1, 3)",
+        "silent_to_asymptomatic_days": "(0, 3, 10)",
+        "silent_to_symptomatic_days": "(0, 3, 10)",
+        "asymptomatic_to_recovered_days": "(3, 5, 7)",
+        "symptomatic_to_asymptomatic_days": "(7, 10, 14)",
+        "symptomatic_to_hospitalized_days": "(0, 1.5, 10)",
+        "hospitalized_to_asymptomatic_days": "(18)",
+        "hospitalized_to_icu_days": "(5)",
+        "icu_to_deceased_days": "(7)",
+        "icu_to_hospitalized_days": "(7)"
+    },
```

Comment:
these should be flattened (no sub-dicts)

Reply:
also calling dist

Reply:
This file is deprecated, I deleted it